src/components/login/LoginPage.js

```diff
@@ -1,85 +1,124 @@
 
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { Component } from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import GoogleButton from '../common/GoogleButton';
 import emailIcon from '../../../public/assets/icons/mail-icon.svg';
+import { globalRequest, globalFailure } from '../../actions/globalActions';
+import login from '../../actions/googleActions';
+import loginGoogle from '../../utils/loginGoogle';
 
-const LoginPage = () => (
-  <div>
-    <div className="site-content vertical-center">
-      <div className="container">
-        <div className="row login-wrapper">
-
-          <div className="col-md-4 signup text-center order-md-2">
-            <div className="row welcome-banner align-items-center">
-              <div className="col-md-12">
-                <h2 className="welcome-text">Welcome back!</h2>
-                <p className="text-center welcome-body">To keep sharing your exciting Ideas please log in into your account</p>
-              </div>
-            </div>
+class LoginPage extends Component {
+  static propTypes = {
```

Comment:
Should you not have this outside the class, just as you have for the `mapstatetoprops`?